uiowa/uiowa
# CCOM split and report an issue link

tracks #6630

This PR adds a `ccom` split and when enabled, it adds a link to the footer after the contact info that links to the report an issue form and pre-populates the URL field.

## To test:
1. Sync a site of your choice
2. observe there is no link
3. `ddev drush @YOURSITE.local config-split:activate ccom -y`
4. Observe there is a link
5. Verify with a member of CCOM that the link creates the properly formatted URL from multiple pages.
